addons/website_crm/migrations/11.0.2.0/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
```

Comment:
The parameter is env by default since v10, so this is should be renamed and put below `env.cr`